apps/ui/src/components/ProposalVoteChoice.vue

```diff
@@ -12,16 +12,26 @@ withDefaults(
     showReason: true
   }
 );
+
+const isClosed = computed(() => {
+  return ['passed', 'executed', 'rejected'].includes(props.proposal.state);
+});
 </script>
 
 <template>
-  <div
-    v-if="!!proposal.privacy && !proposal.completed"
-    class="flex gap-1 items-center"
-  >
-    <span class="text-skin-heading leading-[22px]">Encrypted choice</span>
-    <IH-lock-closed class="size-[16px] shrink-0" />
-  </div>
+  <template v-if="!!proposal.privacy && !proposal.completed">
+    <div v-if="isClosed" class="flex flex-col items-start">
+      <div class="flex gap-1 items-center">
+        <div class="text-skin-heading leading-[22px]">Decrypting choice</div>
+        <IH-lock-closed class="size-[16px] shrink-0" />
+      </div>
+      <div class="text-[17px] text-skin-text">Please wait ...</div>
+    </div>
+    <div v-else class="flex gap-1 items-center">
+      <span class="text-skin-heading leading-[22px]">Encrypted choice</span>
+      <IH-lock-closed class="size-[16px] shrink-0" />
+    </div>
+  </template>
```

Comment:
I think I'd personally just leave old message here as it was. Having lots of `Decrypting choice/Please wait` messages being kind of jammed in there while we display general message on the UI about it being in progress might be excessive (on v1 it looks like we just displayed Encrypted choice there).